lib/cog_api/http/relay_groups.ex

```diff
@@ -38,30 +38,19 @@ defmodule CogApi.HTTP.RelayGroups do
     |> ApiResponse.format_delete("The relay group could not be deleted")
   end
 
-  def update_memberships(action, %{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
-    case get_group_relay(name, relay_name, endpoint) do
-      {:ok, relay, relay_group} ->
-        update_membership(relay_group.id, relay.id, action, endpoint)
-      error ->
-        error
+  def update_memberships(action, %{name: name}, %{relays: relay_names}, endpoint) when is_list(relay_names) do
```

Comment:
@christophermaier I think the major issue actually revolves around how the function is being called. It's dependent on that map with the name key being there. cogctl calls these functions so it's a bit more far reaching. If I can I would like to go ahead and get this merged, but I can see about refactoring next. If that sounds reasonable?